docs/blog/2019-08-06-gazette-august/index.md

```diff
@@ -0,0 +1,124 @@
+
+## Team Highlights
+
+### Snow Leopard Release
+
+The past few weeks has been all about what we're calling the Snow Leopard release (kudos to [Dustin](https://twitter.com/schaudustin) for the name).
+
+First, an aside.
+
+[Mac OS X Snow Leopard](https://en.wikipedia.org/wiki/Mac_OS_X_Snow_Leopard) (version 10.6) was the seventh major release of Mac OS X (now named macOS). Unlike those of previous versions, the goals of Snow Leopard were improved performance, greater efficiency and the reduction of its overall memory footprint. Addition of new end-user features was not a primary consideration: its name signified its goal to be a refinement of the previous OS X version, Leopard.
+
+Similarly, earlier last month, we felt like we needed to take some time to slow down and clean up Gatsby after some of the big features that we've shipped recently ([themes](https://www.gatsbyjs.org/blog/2019-07-03-announcing-stable-release-gatsby-themes/), [per page manifest](https://www.gatsbyjs.org/blog/2019-06-12-performance-improvements-for-large-sites/)). Our goal was to squash bugs and issues that had happened as a result of moving fast and shipping.
+
+We called this the [Snow Leopard](https://github.com/gatsbyjs/gatsby/projects/13) release.
+
+We think this was a good idea because besides being the best React framework there is, we also want to be the most stable one.
+
+Looks like people think we're doing fine. 🙂
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">I have an opinionated starter for creating course sites built with <a href="https://twitter.com/gatsbyjs?ref_src=twsrc%5Etfw">@gatsbyjs</a> that I hadn&#39;t updated since January so I full well expected to spend a few hours fixing it after I updated all the deps.<br><br>Nope, just updated the packages and everything worked. Thanks Gatsby team!</p>&mdash; Brian Holt (@holtbt) <a href="https://twitter.com/holtbt/status/1158559782451609600?ref_src=twsrc%5Etfw">August 6, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+### Jason Blitz Lengstorf
+
+Jason has basically been living his life on livestream since the past month. He's done some incredible streams and has many more exciting ones coming up.
+
+My favourite of the past few ones was [Advanced GraphQL techniques in Gatsby](https://www.twitch.tv/videos/462900728) Mikhail Novikov.
+
+### Theme Jam
+
+To celebrate the launch of Gatsby Themes, we announced the [Gatsby Theme Jam](https://themejam.gatsbyjs.org/) last month.
+
+The response was incredible. Seeing so many wonderful themes from all of you has been spectacular and we're afraid we might like _too_ many of them to pick one.
+
+The team is currently hard at work judging and picking their favourites so results should be out very soon!
+
+## Community Highlights
+
+Gatsby has always been a community effort. We get a lot of contributions every month. Want proof?
```

Comment:
This section could be reversed? Start with thanks, then on to highlights, and finally this tweet. That places the most importance on the thanking, then goes straight in to what's changed, finally summarising with this section.

Let's put the emphasis on you instead of us:

```suggestion
Gatsby has always been a community effort. You make a huge number of contributions every month. In fact we recently had a record breaking month:
```

Reply:
Much better! Thank you